discord/gateway.py

```diff
@@ -830,6 +836,8 @@ def __init__(
         self._keep_alive: Optional[VoiceKeepAliveHandler] = None
         self._close_code: Optional[int] = None
         self.secret_key: Optional[List[int]] = None
+        # defaulting to -1
+        self.seq_ack: int = -1
```

Comment:
Not a fan of `-1` as a sentinel, everything else uses `None` to denote that it isn't set.

Reply:
I see it in the documentation. Kind of ugly but it is what it is.